.github/workflows/daily-builds.yml

```diff
@@ -41,6 +41,12 @@ jobs:
     uses: ./.github/workflows/checkbox-snap-daily-builds.yml
     secrets: inherit
 
+  checkbox-ce-oem-snap-daily:
+    needs: check_for_commits
+    if: ${{ needs.check_for_commits.outputs.new_commit_count > 0 || github.event_name == 'workflow_dispatch' }}
```

Comment:
you need to edit the line 29 of this file adding the path to the provider! else your commits to the provider are not enough to trigger a rebuild. (I would suggest change but github doesn't let me comment on unedited lines)

Basically go from 
```
commit_count=$(git log --since="$LAST_PASS_DAILY_BUILD" --oneline -- checkbox-ng checkbox-support providers checkbox-core-snap checkbox-snap | wc -l)
```
to
```
commit_count=$(git log --since="$LAST_PASS_DAILY_BUILD" --oneline -- checkbox-ng checkbox-support providers checkbox-core-snap checkbox-snap contrib/checkbox-ce-oem | wc -l)
```

Reply:
Thank you! Fixed!